recipe/meta.yaml

```diff
@@ -20,15 +20,23 @@ source:
     # qiskit-aer builds against a bundled blas on Windows. This patch adds a
     # USE_BUNDLED_BLAS_WIN cmake option to search for an installed blas library
     # instead
+    # This patch was accepted in https://github.com/Qiskit/qiskit-aer/pull/1940
+    # and can be removed from the receipe with the 0.14 release.
     - 0001-windows-find-blas.patch
+    # qiskit-aer uses an API from unistd.h to determine the total system memory
+    # at runtime. This works for Linux and for macOS when using the GitHub
+    # Actions C toolchain, but it does not work with the conda-forge C
+    # toolchain. This patch uses a different API. It was submitted upstream in
+    # https://github.com/Qiskit/qiskit-aer/pull/2016.
+    - 0001-Use-Apple-specific-API-to-determine-system-memory-on.patch  # [osx]
 
 build:
   script:
     # cuQuantum only supports Volta+
     - export AER_CUDA_ARCH="7.0;7.5;8.0;8.6;9.0+PTX"  # [cuda_compiler != "None"]
     - CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
   # Skip macOS build until https://github.com/Qiskit/qiskit-aer/issues/1689 is addressed
```

Comment:
Given this PR adds macOS, maybe this comment should be dropped as it is no longer relevant?

```suggestion
```

Reply:
Oops! I'll take that comment out with the next update. My macOS patch was accepted upstream closing the linked issue there, so with the next update we can take out the Windows and macOS patches and that comment.

Reply:
No worries. It happens. Opened issue ( https://github.com/conda-forge/qiskit-aer-feedstock/issues/24 ) to track